packages/core/content-type-builder/server/services/components.js

```diff
@@ -52,6 +52,9 @@ const createComponent = async ({ component, components = [] }) => {
   });
 
   await builder.writeFiles();
+
+  strapi.eventHub.emit('component.create', { component: newComponent });
```

Comment:
You're right, it's something we should discuss to try to be consistent

My thinking was that using an object made room for more data in the future (by adding keys to that object) without breaking changes. Also the existing entity events [kind of look like this](https://github.com/strapi/strapi/blob/7c63d5cdaf5210d73e2e474cfd3d4d74d45143f7/packages/core/strapi/lib/services/entity-service/index.js#L66-L69).

So it's currently not needed but I was thinking it might help later